provision/provision-site.sh

```diff
@@ -135,6 +135,16 @@ function get_primary_host() {
 function vvv_provision_site_nginx_config() {
   local SITE_NAME=$1
   local SITE_NGINX_FILE=$2
+
+  if [ ! -f "$SITE_NGINX_FILE" ]; then
+    vvv_warn " * VVV didn't found any Nginx config file, it will use a fallback config file"
+    cp -f "/srv/provision/core/nginx/config/sites/fallback.conf" "$SITE_NGINX_FILE"
+  fi
+  local DEST_NGINX_FILE=${SITE_NGINX_FILE//\/srv\/www\//}
```

Comment:
after all those time I am not sure if these lines are needed as we have there on https://github.com/Varying-Vagrant-Vagrants/VVV/blob/2593a4cbba2ccdbaeb7f909a4bd5c6296d0224df/provision/provision-site.sh#L181